make/utils_exposed.cmake

```diff
@@ -145,12 +145,45 @@ function (_DR_check_if_linker_is_gnu_gold var_out)
   set(${var_out} ${is_gold} PARENT_SCOPE)
 endfunction (_DR_check_if_linker_is_gnu_gold)
 
+# Takes in a target and returns the expected target name.
+#
+# XXX i#1557: DynamoRIO cmake files used to query the LOCATION target property at
+# configure time. This property has been made obsolete, see CMP0026. The function
+# here can be used to retrieve the default target name instead. However, it only
+# supports the default names, as seen at configure time.
+function(DynamoRIO_get_full_name out target)
```

Comment:
Please rename: `DynamoRIO_get_full_path` or `DynamoRIO_get_absolute_path`

Reply:
DynamoRIO_get_full_output_name ok? It's really not just about the path, but about including the output name

Reply:
Done